rust/entity/src/job.rs

```diff
@@ -0,0 +1,17 @@
+//! `SeaORM` Entity. Generated by sea-orm-codegen 0.11.3
+
+use sea_orm::entity::prelude::*;
+
+#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq)]
+#[sea_orm(table_name = "job")]
+pub struct Model {
```

Comment:
should `Model` here be `Job` or similar? We will eventually have several "Models"